src/bin/utils.ts

```diff
@@ -155,11 +156,50 @@ async function retryAuth<T>(
   }
 }
 
+const envVariables = {
+  getPkPassword: () => process.env.PK_PASSWORD,
+  getPkRecoveryCode: () => process.env.PK_RECOVERY_CODE,
+};
+
+async function getByFile(
+  passwordFile: string | undefined,
+): Promise<string | undefined> {
+  if (passwordFile == null) return;
+  // Reading the file.
+  const contents = await fs.promises.readFile(passwordFile, {
+    encoding: 'utf-8',
+  });
+  return contents.trim();
+}
+
+async function getPassword(passwordFile: string | undefined): Promise<string> {
```

Comment:
All these sorts of functions should be in `bin/options.ts` or `bin/parser.ts`. We had made the movements already in the CARL MR, that was the intention behind these.

Reply:
It's only used in `commandStart.ts` right now for getting the password. CARL isn't used there since it's not a GRPC call and it's not a command option since it's the password and not the passwordFile.

They should be used in the bootstrap command as well.